core/API/ResponseBuilder.php

```diff
@@ -189,6 +189,23 @@ private function handleArray($array)
 
         $isAssoc = !empty($firstArray) && is_numeric($firstKey) && is_array($firstArray) && count(array_filter(array_keys($firstArray), 'is_string'));
 
+        if (is_numeric($firstKey)) {
+            $columns = Common::getRequestVar('filter_column', false, 'array', $this->request);
```

Comment:
`filter_column[]` and `filter_pattern` are now supported if an indexed array is returned